TypeCobol/Compiler/Diagnostics/CodeElementCheckers.cs

```diff
@@ -289,6 +289,32 @@ public static void OnCodeElement(StartStatement statement, CodeElementsParser.St
         }
     }
 
+    class CodeElementChecker
+    {
+        public static void OnCodeElement(CodeElement codeElement, bool debuggingMode)
+        {
+            if (debuggingMode)
+            {
+                // detect CodeElement with a mix of Debug and "Normal" lines in debugging mode
+                int consumedTokensCount = codeElement.ConsumedTokens.Count;
+                if (consumedTokensCount > 1)
+                {
+                    bool isDebug = false, isNoDebug = false;
+                    for (int i = 0; i < consumedTokensCount; i++)
+                    {
+                        bool isDebugType = char.ToLower(codeElement.ConsumedTokens[i].TokensLine.IndicatorChar) == 'd';
+                        isDebug |= isDebugType;
+                        isNoDebug |= !isDebugType;
+                    }
```

Comment:
Why not stopping loop as soon as (isDebug && isNoDebug) is true? I think we can be an operator | is used.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/tree/95744efb6e12dc402f62f7c87de96a65cf496b6a